codegen/templates/main.tmpl

```diff
@@ -9,20 +10,20 @@ import (
 
 	"go.uber.org/zap"
 	"github.com/uber/zanzibar/runtime"
-	{{range $idx, $pkg := .IncludedPackages -}}
-	{{$pkg.AliasName}} "{{$pkg.PackageName}}"
-	{{end}}
+
+	module "{{$instance.PackageInfo.ModulePackagePath}}"
 )
 
 func getDirName() string {
 	_, file, _, _ := runtime.Caller(0)
 	return zanzibar.GetDirnameFromRuntimeCaller(file)
 }
 
+// TODO: remove this
 func getConfigDirName() string {
 	return filepath.Join(
 		getDirName(),
-		"{{.RelativePathToAppConfig}}",
+		"../../../",
```

Comment:
Yes it seems bad, but it was hard coded inside the execution of the template without configuration anyway. I'm punting on this entire form of config being removed because it doesn't work when we purge source and run a binary in its own directory.